lib/commons/text/is-valid-autocomplete.js

```diff
@@ -59,7 +59,8 @@ export const autocomplete = {
     'email',
     'impp'
   ],
-  locations: ['billing', 'shipping']
+  locations: ['billing', 'shipping'],
+  invalidButSafe: ['text', 'pronouns', 'gender', 'message', 'content']
```

Comment:
I think I'd prefer for these to come from the check option. And maybe rename this to `ignoredValues`? Safe doesn't feel quite right here. We incomplete these because we don't know if they're safe, but neither are we certain that they're problematic.


Have a look at checks/aria/aria-allowed-attr.json for an example of check options. The option then needs to be documented in doc/check-options.md.